git-workflows/scripts/detect-conventions.sh

```
@@ -0,0 +1,278 @@
+        --arg confidence "$confidence" \
+        --arg config_file "$config_file" \
+        '{
+          success: true,
+          uses_conventional_commits: $uses_conventional,
+          detection_method: $detection_method,
+          confidence: $confidence,
+          contributing_file: $config_file
+        }'
+    elif [ "$detection_method" = "commit_history" ]; then
+      # Convert match rate to float for jq
+      local match_rate_float=$(awk "BEGIN {printf \"%.2f\", $pattern_match_rate / 100}")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)